phases/01-math-foundations/en-simple/module-a-linear-algebra.md

```
@@ -0,0 +1,1894 @@
+    return np.dot(a, b) / (np.linalg.norm(a) * np.linalg.norm(b))
+
+names = list(words)
+for i in range(len(names)):
+    for j in range(i + 1, len(names)):
+        s = cos_sim(words[names[i]], words[names[j]])
+        print(f"{names[i]:4} vs {names[j]:4}: {s:.3f}")
+```
+
+Results (about):
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)